bytes/src/test/scala/io/iohk/ethereum/utils/ByteStringUtilsTest.scala

```diff
@@ -0,0 +1,21 @@
+package io.iohk.ethereum.utils
+
+import akka.util.ByteString
+import org.scalatest.wordspec.AnyWordSpec
+
+class ByteStringUtilsTest extends AnyWordSpec {
+
+  "ByteStringUtilsTest" should {
+
+    "concatByteStrings for simple bytestrings" in {
+      val bs1 = ByteString("000")
+      val bs2 = ByteString("111")
+      val summarized: ByteString = bs1 ++ bs2
+
+      val concatenated: ByteString = ByteStringUtils.concatByteStrings(bs1, bs2)
+      println(s"Bs1: ${bs1} Bs2: ${bs2} Sum: ${summarized} Concat: ${concatenated}")
```

Comment:
I'm not sure we should keep `println` in the code. If it was just used for debugging it can be removed.